db/migrate/20201124063738_create_addresses.rb

```diff
@@ -0,0 +1,12 @@
+
+class CreateAddresses < ActiveRecord::Migration[5.2]
+  def change
+    create_table :addresses do |t|
+      t.references :ems, :type => :bigint, :index => true, :references => :ext_management_system
+      t.string :ems_ref
+      t.references :physical_storage, :type => :bigint, :index => true
+      t.references :physical_storage_consumer, :type => :bigint, :index => true
```

Comment:
If we plan this to be a general address model that can be used to hold various addresses addresses, then the container of the address should be generic. Not hard-coded to `physical_storage_consumer`. Kinda like in `Endpoint`

`t.belongs_to :resource, :polymorphic => true, :type => :bigint`